portfolio/pom.xml

```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
+          <deploy.projectId>vzapataiii-sps-summer20</deploy.projectId>
```

Comment:
Side note for the future: this is a good example of something that could/should be in a separate PR. I know it might feel a bit silly to do a whole PR for a one line change (it actually ends up being somewhat common). But since it a separate change from the rest it makes sense to split off.